model/project_vars.go

```diff
@@ -556,24 +576,104 @@ func isParameterStoreEnabledForProject(ctx context.Context, projectID string) (b
 		return false, nil
 	}
 
-	projRef, err := FindMergedProjectRef(projectID, "", false)
+	return ref.ParameterStoreEnabled, nil
+}
+
+// findProjectRef finds the project ref associated with the project variables.
+// Returns a bool indicating if it's a branch project ref or a repo ref.
```

Comment:
```suggestion
// Returns a bool indicating if it's a repo ref (true) or a branch project ref (false).
```

Reply:
I think that's already pretty clear because the boolean is called `isRepoRef`, so I don't think it's worth double documenting.